From 2657e6760f9c439b16ef4b498fe1fb284c250cc1 Mon Sep 17 00:00:00 2001
From: Phoenix591 <kcelbert@gmail.com>
Date: Sat, 3 Feb 2018 06:41:08 +0000
Move the earliest valid current time up to trigger Fixup_Time on more devices. For example my 
device, the LG V20 (US996), the default time comes back as 2016 which doesn't trigger the 
fixup code. The actual time I picked was pretty arbitrary. 

Change-Id: I01db64777d7d0c5ce0710445cf88858d89656ac0
---
 twrp-functions.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/twrp-functions.cpp b/twrp-functions.cpp
index b7bcebe2..bc09541e 100644
--- a/twrp-functions.cpp
+++ b/twrp-functions.cpp
@@ -904,7 +904,7 @@ void TWFunc::Fixup_Time_On_Boot(const string& time_paths /* = "" */)
 
 		gettimeofday(&tv, NULL);
 
-		if (tv.tv_sec > 1405209403) { // Anything older then 12 Jul 2014 23:56:43 GMT will do nicely thank you ;)
+		if (tv.tv_sec > 1517600000) { // Anything older then 2 Feb 2018 19:33:20 GMT will do nicely thank you ;)
 
 			LOGINFO("TWFunc::Fixup_Time: Date and time corrected: %s\n", TWFunc::Get_Current_Date().c_str());
 			fixed = true;
-- 
2.16.1

